gtsfm/scene_optimizer.py

```diff
@@ -117,7 +117,10 @@ def create_computation_graph(
         keypoints_graph_list = []
         descriptors_graph_list = []
         for delayed_image in image_graph:
-            (delayed_dets, delayed_descs) = self.feature_extractor.create_computation_graph(delayed_image)
+            (
```

Comment:
it looks like this whole file has been reformatted without the 120 character limit, did you change that already?

Reply:
I dont see any code changes here, just reformatting that may not be intentional.

Reply:
I've formatted all of these files with `black --line-length=120 filepath`. I didn't see mention of additional settings in the style guide - is there additional black configuration to get the formatting you're expecting?